bitbucket/client.py

```diff
@@ -1,12 +1,13 @@
 import requests
+import json
 
 from bitbucket.exceptions import UnknownError, InvalidIDError, NotFoundIDError, NotAuthenticatedError, PermissionError
 
 
 class Client(object):
     BASE_URL = 'https://api.bitbucket.org/'
 
-    def __init__(self, user, password, owner=None):
+    def __init__(self, user=None, password=None, token=None):
```

Comment:
extend the parameters to include client_secret and cleint_id as well as optional parameters.